gplugins/lumerical/utils.py

```diff
@@ -0,0 +1,154 @@
+from gdsfactory.technology import LayerStack
+from gdsfactory.typings import PathType
+from gdsfactory.pdk import get_layer_stack
+from gdsfactory.config import logger
+from xml.etree.ElementTree import Element, SubElement
+from xml.dom import minidom
+import xml.etree.ElementTree as ET
+from pathlib import Path
+
+um = 1e-6
+
+
+def to_lbr(
```

Comment:
to_lbr would make sense if this were a method of MaterialMap, maybe material_map_to_lbr ?

Reply:
layerstack_to_lbr seems more appropriate since it is more broad. The lbr file encompasses all info about the layerstack.